eng/common/instructions/azsdk-tools/validate-service-label.instructions.md

```diff
@@ -6,22 +6,42 @@ tools: ['azsdk_check_service_label', 'azsdk_create_service_label']
 ## Goal
 Validate service label exists or create new one for SDK release process.
 
-## Step 1: Get Service Label
-Ask user for their service label. If none provided, explain that a valid service label is required for SDK release.
+## Step 1: Provide Information
+
+Provide the following information about the importance of service labels:
+
+"Before your SDK is released, your service must have a valid service label. Service labels enable automatic owner assignment and notifications across the Azure SDK ecosystem.
+
+When properly configured, service labels automatically:
+
+- Notify service owners when issues are filed against their SDK
+- Add appropriate reviewers to pull requests
+- Connect code changes to the right team members through CODEOWNERS integration"
+
+## Step 2: Get Service Label
+
+Ask user for their service label. If they don't know their service label provide guidance:
+
+- Check out the [Common Labels CSV](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/data/common-labels.csv) file and look for a row whose first column contains your service's product name.
```

Comment:
Should this be quoted as well like the above snippets we expect to be sent to the user?